changelog.md

```diff
@@ -4,12 +4,12 @@
 
 - Apply flake8 and numpy docstrings to limiter & protocol (@lwasser, #326)
 - Update client's stream method to warn when using unofficial parameters (@enadeau, #385)
-- Type annotation to client file (@enadeau, #384)
 - Fix docstring in SleepingRateLimitRule (@enadeau)
 - Fix: rename `SubscriptionCallback.validate` -> `SubscriptionCallback.validate_token` to avoid conflict with `pydantic.BaseModel` (@lwasser, #394)
 - Fix: docstrings in model.py, documentation errors, findfonts warning suppression by removing opengraph (temporarily), typing updates (@lwasser, #387)
 - Fix: read the docs is breaking due to pydantic json warnings, also update python version on build and sync pr previews (@lwasser, #412)
 - Fix: update master to main in all builds (@lwasser)
+- Type annotation for all file in the library (@enadeau, #384, #415)
```

Comment:
🎉 🚀

Reply:
amazing work! i haven't looked at the files yet just responding to the NEP-29 / pint / python 3.8 comment below!!